airbyte-webapp/scripts/dev-overwrites.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+const path = require("path");
+
+const EXPERIMENTS_FILE = path.resolve(__dirname, "../.experiments.json");
+
+if (fs.existsSync(EXPERIMENTS_FILE)) {
+  console.log("\nOverwriting experiments from .experiments.json ...");
```

Comment:
Just a note: these `console.log` calls get cleared by CRA in the console very quickly, so not _super_ useful. With vite we could disable the `clearScreen` option to keep this debug info in the console.